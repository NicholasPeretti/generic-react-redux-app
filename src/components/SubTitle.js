import React from 'react'
import styled from 'styled-components'

const StyledH3 = styled.h3`
  margin: 0px;
  font-size: 18px;
  font-weight: normal;
`

const SubTitle = ({ children, style = {} }) => (
  <StyledH3 style={style}>
    {children}
  </StyledH3>
)

export default SubTitle
