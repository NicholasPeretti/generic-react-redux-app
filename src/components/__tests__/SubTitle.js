import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import SubTitle from '../SubTitle'

describe('SubTitle', () => {
  PresentationalTest(SubTitle)

  test('Should match snapshot', () => {
    const wrapper = shallow(<SubTitle>This is a SubTitle</SubTitle>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
