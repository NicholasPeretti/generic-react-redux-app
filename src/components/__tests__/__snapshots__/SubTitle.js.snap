// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle Should match snapshot 1`] = `"<h3 class=\\"sc-bdVaJa fkyhBf\\">This is a SubTitle</h3>"`;
